src/js/node/http.ts

```diff
@@ -1295,7 +1300,13 @@ class ClientRequest extends OutgoingMessage {
         verbose: !!$debug,
         signal: this[kAbortController].signal,
         proxy: proxy,
-
+        tls: {
+          ca: this.#ca,
```

Comment:
does nodejs support `passphrase`, `dhparam`, `ciphers`  and `secureOptions` here? If it does we should also support it.

Reply:
`dhparam` does not seam to be supported but ive added the others which node supports and he have to